dart-impl/mpi/src/dart_communication.c

```diff
@@ -30,6 +30,11 @@
 #include <math.h>
 #include <alloca.h>
 
+/* the number of consecutive memcpy for local put/get before calling into MPI */
+#define NUM_CONSECUTIVE_MEMCPY 16
```

Comment:
why 16 ? educated guess?

Reply:
Yes, I'm open to other suggestions :D

Reply:
at least document, that 16 is just an educated guess